LUX/Classes/Utilities/Functions/BindViewModel.swift

```diff
@@ -15,3 +17,9 @@ public func bindButtonEnabledToPublisher(_ button: UIButton, publisher: AnyPubli
 public func bindActivityIndicatorVisibleToPublisher(_ activity: UIActivityIndicatorView, publisher: AnyPublisher<Bool, Never>, cancelBag: inout Set<AnyCancellable>) {
     publisher.sink{ activity.isHidden = !$0 }.store(in: &cancelBag)
 }
+
+extension Publisher where Output == Bool , Failure == Never {
+    public func bind<T>(keyPath: WritableKeyPath<T, Bool>, value: T, cancelBag: inout Set<AnyCancellable>) {
```

Comment:
Let's make this read a little easier; maybe something like:

```suggestion
    public func bind<T>(to keyPath: WritableKeyPath<T, Bool>, on value: T, storingIn cancelBag: inout Set<AnyCancellable>) {
```

So that it reads:

```swift
subject.bind(to: \UIButton.isEnabled, on: button, storingIn: &cancelBag)
```